development/playbooks/deploy-dev/deploy-dev.yaml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Deploy Foreman Development Environment
+  hosts: "{{ target_host if target_host is defined and target_host != '' else 'quadlet' }}"
+  become: true
+  vars_files:
+    - "../../../src/vars/defaults.yml"
+    - "../../../src/vars/{{ certificate_source }}_certificates.yml"
+    - "../../../src/vars/images.yml"
+    - "../../../src/vars/database.yml"
+    - "../../../src/vars/foreman.yml"
+    - "../../../src/vars/base.yaml"
+  vars:
+    httpd_foreman_backend: "http://localhost:3000"
+  roles:
+    - pre_install
+    - role: certificates
+    - role: postgresql
+      vars:
+        postgresql_databases:
+          - name: "{{ candlepin_database_name }}"
+            owner: "{{ candlepin_database_user }}"
+          - name: "foreman_development"
+            owner: "{{ foreman_database_user }}"
+          - name: "foreman_development_test"
+            owner: "{{ foreman_database_user }}"
+          - name: "{{ pulp_database_name }}"
+            owner: "{{ pulp_database_user }}"
+        postgresql_users:
+          - name: "{{ candlepin_database_name }}"
+            password: "{{ candlepin_database_password }}"
+          - name: "{{ foreman_database_name }}"
```

Comment:
I've found that Foreman needs to be able to recreate the test DB so in practice it needed permissions to do so. Not sure if there's a better way to make them the owner.